helpful.el

```diff
@@ -599,27 +614,25 @@ overrides that to include previously opened buffers."
           (if (stringp raw-source)
               (read raw-source)
             raw-source))
-         (syms (helpful--callees source)))
-    (setq syms
-          (--sort
-           (string< (symbol-name it) (symbol-name other))
-           syms))
+         (syms (helpful--callees source))
+         (primitives (-filter (lambda (sym) (helpful--primitive-p sym t)) syms))
```

Comment:
FWIW I'd be tempted to write this as `(--filter  (helpful--primitive-p it t) syms)`. No big deal though :)